adobe_vipm/flows/fulfillment/change.py

```diff
@@ -54,6 +55,20 @@ class GetReturnableOrders(Step):
     def __call__(self, client, context, next_step):
         adobe_client = get_adobe_client()
         returnable_orders_count = 0
+
+        last_two_weeks = (
+            datetime.fromisoformat(context.adobe_customer["cotermDate"])
+            - timedelta(days=13)
+        ).date()
+
+        if date.today() >= last_two_weeks:
+            logger.info(
+                "Downsize occurs in the last two weeks before the anniversary date. "
```

Comment:
Since this is going to be in logs, can you please put more details here. Like Returnable orders are not going to be submitted,